sdk/servicebus/azure-servicebus/stress/templates/network_loss.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: chaos-mesh.org/v1alpha1
-kind: NetworkChaos
+kind: Schedule
```

Comment:
If you don't mind while you're in this file, can you update it to use the new [chaos wrapper](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_chaos.tpl)? This should replace the metadata and annotations with the addons boilerplate version for consistency.

Reply:
Also @richardpark-msft @m-redding FYI the chaos resources are migrating to the `Schedule` kind going forward, since that's how it's structured in chaos mesh 2.x